app/bundles/LeadBundle/Command/UpdateLeadListsCommand.php

```diff
@@ -56,18 +56,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         if ($id) {
             $list = $listModel->getEntity($id);
+
             if ($list !== null) {
-                $output->writeln('<info>'.$translator->trans('mautic.lead.list.rebuild.rebuilding', ['%id%' => $id]).'</info>');
-                $processed = 0;
-                try {
-                    $processed = $listModel->rebuildListLeads($list, $batch, $max, $output);
-                } catch (QueryException $e) {
-                    $this->getContainer()->get('monolog.logger.mautic')->error('Query Builder Exception: '.$e->getMessage());
+                if ($list->isPublished()) {
```

Comment:
Should we not include the previous `!== null` statement here as well?

Reply:
@Enc3phale

Reply:
@Woeler sorry for the delay.
Which statement do you want to add?
We are inside `if ($list !== null) {` it's not necessary to write `if ($list !== null && $list->isPublished()) {`.

Reply:
Ugh you're right, I'm reading like a dumbass. Good to go. Gonna merge this, sorry!